src/main/java/dev/jbang/source/ScriptSource.java

```diff
@@ -16,22 +39,14 @@
 import java.util.Properties;
 import java.util.Set;
 import java.util.function.Function;
+import java.util.function.Predicate;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import dev.jbang.Cache;
-import dev.jbang.Settings;
-import dev.jbang.cli.BaseCommand;
-import dev.jbang.cli.ExitException;
-import dev.jbang.dependencies.DependencyUtil;
-import dev.jbang.dependencies.Detector;
-import dev.jbang.dependencies.MavenRepo;
-import dev.jbang.dependencies.ModularClassPath;
-import dev.jbang.util.JavaUtil;
-import dev.jbang.util.PropertiesValueResolver;
-import dev.jbang.util.Util;
+import static dev.jbang.cli.BaseBuildCommand.createJarFile;
+import static dev.jbang.cli.BaseBuildCommand.resolveInJavaHome;
```

Comment:
same as above.

Reply:
moving `resolveInJavaHome` to `JdkManager`